src/main/webapp/dev/js/common/const.js

```diff
@@ -88,6 +88,7 @@ const ParamsNames = {
     FEEDBACK_QUESTION_NUMSCALE_STEP: 'numscalestep',
     FEEDBACK_QUESTION_RANKOPTION: 'rankOption',
     FEEDBACK_QUESTION_RANKTORECIPIENTS: 'rankToRecipients',
+    FEEDBACK_QUESTION_NUMBER: 'questionnum',
```

Comment:
Declaring this here as a common variable looks fine as `Const.java` has a similar parameter named `FEEDBACK_QUESTION_NUMBER`.

However, refer to my other comment.